src/Console/Test/console_main.cpp

```diff
@@ -51,14 +51,12 @@ void registerExampleCommands(FOEDAG::Session* session) {
                          const char* argv[]) -> int {
     Q_UNUSED(clientData)
     if (argc < 2) return TCL_ERROR;
-    QDir path{argv[1]};
-    QString fileName{"gui_console_proc.tcl"};
-    if (!path.exists(fileName)) return TCL_ERROR;
-    QString fullPath = path.filePath(fileName);
-    FOEDAG::TclConsoleWidget* console = FOEDAG::Init(interp);
-    QString res = console->getPrompt() + "source " + fullPath +
-                  "\nHello world\n" + console->getPrompt();
-    CHECK_EXPECTED("source " + fullPath, res)
+    QFile file(argv[1]);
+    QString expected(argv[2]);
+    if (!file.exists()) return TCL_ERROR;
```

Comment:
if file not exists test will pass successfully.

Reply:
exit (1) in that case. Please add it.
I checked that if the Tcl code fails or there is a diff, the test fails and the CI fails.

Reply:
it still not solve the problem for the whole project. Everyone will expect that **return TCL_ERROR** will cause the CI fail. But it is not. That was the idea of the issue.